server/src/db/migrations/20240501151512_createVotesTable.cjs

```diff
@@ -0,0 +1,24 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+  return knex.schema.createTable("votes", (table) => {
+    table.bigIncrements("id")
+    table.bigInteger("userId").notNullable().unsigned().index().references("users.id")
+    table.bigInteger("reviewId").notNullable().unsigned().index().references("reviews.id")
+    table.integer("voteStatus").notNullable().defaultTo(0)
```

Comment:
ideally rename to "status" rather than `voteStatus`. It is redundant in this case

Reply:
We originally had as 'status', but ran into some issues in certain spots where 'status' was a reserved variable. And we also had some bugs where we were switching between calling it 'status' in some places and 'voteStatus' in others.